apps/web/modules/ui/components/badge/index.tsx

```diff
@@ -1,54 +1,37 @@
+import { type VariantProps, cva } from "class-variance-authority";
+import * as React from "react";
 import { cn } from "@formbricks/lib/cn";
 
-interface BadgeProps {
-  text: string;
-  type: "warning" | "success" | "error" | "gray";
-  size: "tiny" | "normal" | "large";
-  className?: string;
-}
-
-export const Badge: React.FC<BadgeProps> = ({ text, type, size, className }) => {
-  const bgColor = {
-    warning: "bg-amber-100",
-    success: "bg-emerald-100",
-    error: "bg-red-100",
-    gray: "bg-slate-100",
-  };
-
-  const borderColor = {
-    warning: "border-amber-200",
-    success: "border-emerald-200",
-    error: "border-red-200",
-    gray: "border-slate-200",
-  };
+const badgeVariants = cva(
+  "inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2",
+  {
+    variants: {
+      variant: {
+        success: "bg-emerald-100 border-emerald-200 text-emerald-800",
```

Comment:
Can we follow thr variants and styling, that comes with shadcn's badge component by default